puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRpmNightlyRelease.groovy

```diff
@@ -0,0 +1,31 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage ("Build Gem") {
+            steps {
+                dir('gem-build') {
+                    git(url: "https://github.com/${env.OWNER_REPO}", branch: env.BRANCH)
```

Comment:
Are these reference as `env.` or `params.` ?

Reply:
`env.OWNER_REPO` is accessing the `OWNER_REPO` environment variable.

Reply:
oh! `${params.OWNER_REPO}` is also a thing that would work! i didn't realize that!